hadoop-hdfs-project/hadoop-hdfs/src/site/markdown/HDFSCommands.md

```diff
@@ -304,6 +306,7 @@ Usage:
 | `-idleiterations` \<iterations\> | Maximum number of idle iterations before exit. This overwrites the default idleiterations(5). |
 | `-runDuringUpgrade` | Whether to run the balancer during an ongoing HDFS upgrade. This is usually not desired since it will not affect used space on over-utilized machines. |
 | `-asService` | Run Balancer as a long running service. |
+| `-sortTopNodes` | Whether to run the balancer during will sort datanodes based on the utilization so that highly utilized datanodes get scheduled first. |
```

Comment:
What does this sentence mean? It makes me confused.

Reply:
Hi @zhangshuyan0 sir, it means setting the option "-sortTopNodes", each  iteration will choose the high used capacity datanode in the overUtilized list as the source datanode of the moving block.

Reply:
Should it be "Sort datanodes based on the utilization so that highly utilized datanodes are scheduled first." ?

Reply:
Update PR.
@zhangshuyan0 please help review it again, Thanks.